school/service.go

```diff
@@ -67,32 +65,35 @@ func (s *Service) UnmarshalJSON(b []byte) error {
 	*s = Service(t.temp)
 
 	// Extract the links to other entities for later
-	s.chassis = string(t.Links.Chassis)
-	s.managers = string(t.Links.Managers)
-	s.taskService = string(t.Links.TaskService)
-	s.sessionService = string(t.Links.SessionService)
-	s.storageServices = string(t.Links.StorageServices)
-	s.storageSystems = string(t.Links.StorageSystems)
-	s.accountService = string(t.Links.AccountService)
-	s.eventService = string(t.Links.EventService)
-	s.registries = string(t.Links.Registries)
-	s.systems = string(t.Links.Systems)
-	s.compositionService = string(t.Links.CompositionService)
+	s.chassis = string(t.Chassis)
```

Comment:
Good catch. It looks like the Redfish-Interface-Emulator code currently has a bug where it has these properties under Links, while according to the ServiceRoot spec they should be root properties. I've filed https://github.com/DMTF/Redfish-Interface-Emulator/issues/67 to fix this in the emulator code.